### DISCUSSION TITLE
Request for API Documentation on Certificates and Provisioning Profiles

### DISCUSSION THREAD

**Discussion by DeepakAphro121** [codemagic-ci-cd/codemagic-docs#2557] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

Dear Codemagic Support Team,

I am currently using Codemagic for CI/CD and I have noticed that the API documentation does not include endpoints for generating, uploading, and deleting certificates and provisioning profiles.

Could you please provide the necessary APIs or direct me to the relevant documentation for these operations?

Thank you for your assistance.

Best regards,
Deepak Kumar

**Comment by himesh-cm** [comment]
Discussion is being continued here - https://github.com/orgs/codemagic-ci-cd/discussions/2576#discussioncomment-9679321
